core/src/db_manager.cpp

```diff
@@ -26,7 +26,9 @@ DBManager::DBManager() :
         qCritical() << "Database Error:" << db.lastError();
     db_version = checkVersion();
     QSqlQuery create_ban_table("CREATE TABLE IF NOT EXISTS bans ('ID' INTEGER, 'IPID' TEXT, 'HDID' TEXT, 'IP' TEXT, 'TIME' INTEGER, 'REASON' TEXT, 'DURATION' INTEGER, 'MODERATOR' TEXT, PRIMARY KEY('ID' AUTOINCREMENT))");
+    create_ban_table.exec();
```

Comment:
does this actually run the query?

Reply:
As far as I can tell it does absolutely nothing different besides make the compiler happy.